### PR TITLE
Installed Apollo sever and NoDemon.  Created start and start-dev scripts.

### PR DESCRIPTION
**Pull request by merecka**
…pts.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,9 @@
   "description": "Alex's GraphQL project",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
```
Comment (ahaly92) [inline package.json]: this test isnt doing anything. If code isnt doing anything it should be pulled out

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    hello: () => "world"
+  }
+};
+
+const server = new ApolloServer({
+  typeDefs,
+  resolvers
+});
+
+server.listen().then(({ url }) => {
```
Comment (ahaly92) [inline src/index.js]: i would specify a port here. The post should pull from a .env file. This is a good opportunity to pull in dotenv. Its a lot easier to setup in node

File: package.json (hunk 1 above)
Comment (merecka) [reply package.json]: Done

File: src/index.js (hunk 2 above)
Comment (merecka) [reply src/index.js]: Done

**Pull request merged**